vendor/libraries/usabilitydynamics/lib-wp-list-table/lib/classes/class-wp-list-table.php

```diff
@@ -537,7 +537,7 @@ protected function parse_query( $query ){
                   $_query[ 'meta_query' ] = array();
                 }
 
-                if( is_array( $q['value'] ) && !in_array( $map['compare'], array( 'NOT IN', 'IN' ) ) ) {
+                if( is_array( $q['value'] ) && !in_array( $map['compare'], array( 'NOT IN', 'IN' ) ) && 'BETWEEN' != $map['compare'] ) {
                   $map['compare'] = 'IN';
```

Comment:
@mehulkaklotar yes, please add changes to `UsabilityDynamics/lib-wp-list-table` library so we will not lose the data on composer update. I will share the library with you.